<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="8058" started-at="2019-12-04T07:07:29Z" finished-at="2019-12-04T07:07:37Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="8058" started-at="2019-12-04T07:07:29Z" finished-at="2019-12-04T07:07:37Z">
      <class name="com.employeeapi.testcases.TC004_Update_Employee">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.employeeapi.testcases.TC004_Update_Employee@685cb137]" name="setup" is-config="true" duration-ms="112" started-at="2019-12-04T07:07:29Z" finished-at="2019-12-04T07:07:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="updateemployee()[pri:0, instance:com.employeeapi.testcases.TC004_Update_Employee@685cb137]" name="updateemployee" is-config="true" duration-ms="7896" started-at="2019-12-04T07:07:30Z" finished-at="2019-12-04T07:07:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateemployee -->
        <test-method status="FAIL" signature="checkResponseBody()[pri:0, instance:com.employeeapi.testcases.TC004_Update_Employee@685cb137]" name="checkResponseBody" duration-ms="18" started-at="2019-12-04T07:07:37Z" finished-at="2019-12-04T07:07:37Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at com.employeeapi.testcases.TC004_Update_Employee.checkResponseBody(TC004_Update_Employee.java:51)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkResponseBody -->
        <test-method status="PASS" signature="checkStatusCode()[pri:0, instance:com.employeeapi.testcases.TC004_Update_Employee@685cb137]" name="checkStatusCode" duration-ms="2" started-at="2019-12-04T07:07:37Z" finished-at="2019-12-04T07:07:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusCode -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.employeeapi.testcases.TC004_Update_Employee@685cb137]" name="teardown" is-config="true" duration-ms="1" started-at="2019-12-04T07:07:37Z" finished-at="2019-12-04T07:07:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.employeeapi.testcases.TC004_Update_Employee -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
